### PR TITLE
Update PERPHeads presence.ts

### PR DESCRIPTION
**Pull request by Jackelele**
Fixed errors/issues with code

### CODE REVIEW

**Review by Bas950 — CHANGES_REQUESTED** [review]
Bump version, upload proof of it working

**Comment by Jackelele** [comment]
image

image

File: websites/P/PERPHeads/presence.ts [hunk 1]
```diff
@@ -46,10 +43,7 @@ presence.on("UpdateData", async () => {
       title = document.querySelector(
         "div.p-body > div.p-body-inner > div.p-body-header > div.p-title > h1"
       );
-      search = document.querySelector(
-        "div.p-body > div.p-body-inner > div.p-body-header > div.p-title > h1"
-      );
-      presenceData.state  = title.innerText.replace(search.innerText, "").replace("»", "");
+      presenceData.state  = title.innerText;
```
Comment (ririxidev) [inline websites/P/PERPHeads/presence.ts:46]: 2 spaces instead of one

File: websites/P/PERPHeads/presence.ts [hunk 2]
```diff
@@ -24,10 +24,7 @@ presence.on("UpdateData", async () => {
...
```
Comment (ririxidev) [inline websites/P/PERPHeads/presence.ts:27]: 2 spaces instead of one

File: websites/P/PERPHeads/presence.ts [hunk 3]
```diff
@@ -133,10 +127,7 @@ presence.on("UpdateData", async () => {
...
```
Comment (ririxidev) [inline websites/P/PERPHeads/presence.ts:130]: 2 spaces instead of one

**Review by ririxidev — CHANGES_REQUESTED** [review]
bump version

File: websites/P/PERPHeads/presence.ts (hunk 3 above)
Comment (Jackelele) [reply websites/P/PERPHeads/presence.ts]: Fixed all these now and bumped version :)

**Review by ririxidev — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]
Bump version, upload proof of it working

**Pull request merged**